content/tutorials/sweeps.md

```diff
@@ -6,219 +6,166 @@ menu:
+## Step 1: Define a sweep
 
+A W&B Sweep combines a strategy for testing hyperparameter values with code that evaluates them. Define your sweep strategy with a _sweep configuration_ before starting the sweep.
 
 {{% alert %}}
-The sweep configuration you create for a sweep must be in a nested dictionary if you start a sweep in a Jupyter Notebook.
-
-If you run a sweep within the command line, you must specify your sweep config with a [YAML file]({{< relref "/guides/models/sweeps/define-sweep-configuration" >}}).
+For sweeps in a Jupyter Notebook, the sweep configuration must be in a nested dictionary. For command-line sweeps, use a [YAML file]({{< relref "/guides/models/sweeps/define-sweep-configuration" >}}).
 {{% /alert %}}
 
 ### Pick a search method
 
-First, specify a hyperparameter search method within your configuration dictionary. [There are three hyperparameter search strategies to choose from: grid, random, and Bayesian search]({{< relref "/guides/models/sweeps/define-sweep-configuration/sweep-config-keys/#method" >}}).
-
-For this tutorial, you will use a random search. Within your notebook, create a dictionary and specify `random` for the `method` key. 
-
+Specify a hyperparameter search method in your configuration dictionary. Choose from grid, random, and Bayesian search. Use random search in this tutorial. Create a dictionary in your notebook and set `method` to `random`.
 
 ```
 sweep_config = {
     'method': 'random'
-    }
+}
 ```
 
-Specify a metric that you want to optimize for. You do not need to specify the metric and goal for sweeps that use random search method. However, it is good practice to keep track of your sweep goals because you can refer to it at a later time.
-
+Specify a metric to optimize. Although not required for random search, it's good practice.
 
 ```
 metric = {
     'name': 'loss',
-    'goal': 'minimize'   
-    }
+    'goal': 'minimize'
+}
 
 sweep_config['metric'] = metric
 ```
 
-### Specify hyperparameters to search through
-
-Now that you have a search method specified in your sweep configuration, specify the hyperparameters you want to search over.
-
-To do this, specify one or more hyperparameter names to the `parameter` key and specify one or more hyperparameter values for the `value` key.
+### Specify hyperparameters to search
 
-The values you search through for a given hyperparamter depend on the type of hyperparameter you are investigating.  
-
-For example, if you choose a machine learning optimizer, you must specify one or more finite optimizer names such as the Adam optimizer and stochastic gradient dissent.
+After setting a search method, specify the hyperparameters to search. Assign hyperparameter names to `parameter` and their possible values to `value`. Values depend on the hyperparameter type.
 
+For example, when choosing a machine learning optimizer, specify optimizer names like Adam and stochastic gradient descent.
```

Comment:
? I think Adam is a specific optimizer and "stochastic gradient dissent" is another.